lib/FundingProgrammes.php

```diff
@@ -30,37 +30,53 @@ public function transform(Entry $entry)
         list('entry' => $entry, 'status' => $status) = EntryHelpers::getDraftOrVersionOfEntry($entry);
         $commonFields = ContentHelpers::getCommonFields($entry, $status, $this->locale);
 
-        return array_merge($commonFields, [
-            'isArchived' => $commonFields['status'] === 'expired' && $entry->legacyPath !== null,
-            'description' => $entry->programmeIntro ?? null,
-            'footer' => $entry->outroText ?? null,
-            'thumbnail' => ContentHelpers::getFundingProgrammeThumbnailUrl($entry),
-            'thumbnailNew' => ContentHelpers::getFundingProgrammeThumbnailUrlNew($entry),
-            'trailImage' => self::buildTrailImage($entry->heroImage->one()),
-            'trailImageNew' => self::buildTrailImage(Images::extractNewHeroImageField($entry->hero)),
-            'contentSections' => array_map(function ($block) {
-                return [
-                    'title' => $block->programmeRegionTitle,
-                    'body' => $block->programmeRegionBody,
+        if ($entry->type->handle === 'fundingProgrammes') {
```

Comment:
Not in love with this, feels like this should be two different transformers and this check could happen at a higher level.

Reply:
Good call, have changed it (so hopefully you'll fall in ❤️ )